src/mixins/poolLaneCrownConfig.js

```diff
@@ -0,0 +1,89 @@
+export default {
+  props: [
+    'node',
+    'graph',
+  ],
+  computed: {
+    isLane() {
+      return this.node.type === 'processmaker-modeler-lane';
+    },
```

Comment:
If we could reverse the inheritance this would not be necessary I think.

Reply:
Unfortunately. I think it contains logic the pool needs to add lanes into it.

Reply:
Shouldn't this be on the pool or the lane then? I don't see how this logic is related to the crown config but I may be missing something.